alphabet-Order-task.js

```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję alphabetOrder(), która zwróci podany łańcuch znaków z literami
+// uporządkowanymi według alfabetu („wyraz” - „arwyz”). Nie przejmuj się na razie polskimi
+// znakami :)
+
+function alphabetOrder(word) {
+    const newWord = word.split("").sort().join("");
```

Comment:
No need for these two lines. Just return the value without assigning it to any variable.